docs/01-start-here/05-development-tips.md

```diff
@@ -19,6 +19,16 @@ You can run the Jasmine unit test suite with
 make test
 ```
 
+### Things to do before you commit
+```
+make fix
```

Comment:
can we add `make fix-commits` in here? it's `fix` but only on committed files in your branch and it's super useful because it's blazingly fast compared to `fix`.

My workflow in this repo now normally is to commit, then fix-commits, then amend the commit.

Reply:
sure, go for it, put whatever you think would be useful as honestly I don't have a lot of idea I just went by what I was told!